packages/start/src/config/index.ts

```diff
@@ -88,9 +90,9 @@ function mergeSsrOptions(options: Array<vite.SSROptions | undefined>) {
   return ssrOptions
 }
 
-export function defineConfig(
+export async function defineConfig(
```

Comment:
We should probably confirm with @nksaraf that having the Vinxi export be a Promise that needs to be resolved won't break in the future.

Reply:
Yeah

Reply:
I think we're good.